DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/CourseContentController.cs

```diff
@@ -19,7 +19,7 @@
     [SetDlsSubApplication(nameof(DlsSubApplication.TrackingSystem))]
     [SetSelectedTab(nameof(NavMenuTab.CourseSetup))]
     [Route("/TrackingSystem/CourseSetup/{customisationId:int}/Content")]
-    [ServiceFilter(typeof(VerifyAdminUserCanAccessCourse))]
+    [ServiceFilter(typeof(VerifyAdminUserCanManageCourse))]

```

Comment:
Should this be limited to those that can "manage"?
Would those admins that can "access/view" need to be able to see the content but not necessairly be able to edit it?
Same applies for similar pages where you see but not change data.

Reply:
I don't think we want to allow users to view the All Centres courses in any of these pages under the CourseSetup tab, I think they're only supposed to be for a centre's own courses.
I think it's safer to assume that we keep the existing behaviour in this MR, then create a new ticket to assess which version we want for each page that has them.